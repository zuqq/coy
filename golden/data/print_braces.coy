fn main() -> () {
    println!("Braces can be escaped by doubling them up: {{}}. Here's a number: {}", 1)
}
