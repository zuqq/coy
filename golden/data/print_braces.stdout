Braces can be escaped by doubling them up: {}. Here's a number: 1
